egs2/musdb18/enh1/local/data.sh

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env bash
+
+# Copyright 2023  Shanghai Jiao Tong University (Authors: Wangyou Zhang)
+# Apache 2.0
+set -e
+set -u
+set -o pipefail
+
+log() {
+  local fname=${BASH_SOURCE[1]##*/}
+  echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+help_message=$(cat << EOF
+Usage: $0 [--use_hq <true/false>] [--sample_rate <44.1k>] [--nchannels <1/2>]
+  optional argument:
+    [--use_hq]: whether to download (uncompressed) high-quality audios (Default: false)
+        Note: Samples in musdb18hq and musdb18 have slightly different durations.
+    [--sample_rate]: sampling rate of the data (Default: 44.1k)
+    [--nchannels]: number of channels of the data (Default: 2)
+EOF
+)
+
+. ./db.sh
+. ./path.sh
+
+use_hq=false
+sample_rate=44.1k
+nchannels=2
+dev_names=(
+    "Actions - One Minute Smile"
+    "Clara Berry And Wooldog - Waltz For My Victims"
+    "Johnny Lokke - Promises & Lies"
+    "Patrick Talbot - A Reason To Leave"
+    "Triviul - Angelsaint"
+    "Alexander Ross - Goodbye Bolero"
+    "Fergessen - Nos Palpitants"
+    "Leaf - Summerghost"
+    "Skelpolu - Human Mistakes"
+    "Young Griffo - Pennies"
+    "ANiMAL - Rockshow"
+    "James May - On The Line"
+    "Meaxic - Take A Step"
+    "Traffic Experiment - Sirens"
+)
+
+stage=0
+stop_stage=100
+
+. utils/parse_options.sh
+
+if [ $# -ne 0 ]; then
+    echo "${help_message}"
+    exit 1;
+fi
+
+if [ -z "${MUSDB18}" ]; then
+    log "Fill the value of 'MUSDB18' of db.sh"
+    exit 1
+fi
+
+if [ $nchannels -eq 1 ]; then
+    log "Use single channel audios"
+elif [ $nchannels -eq 1 ]; then
```

Comment:
It should be `-eq 2`, right?

Reply:
Oh thanks for noticing that! I have fixed it.